ansible/macrobench.yml

```diff
@@ -35,7 +38,7 @@
   tags:
     - prometheus
   roles:
-    - { role: cloudalchemy.prometheus, when: stats_remote_db_host is defined }
+    - { role: cloudalchemy.prometheus, when: stats_remote_db_host is defined and arewefastyet_last_exec_is_same is undefined }
```

Comment:
There is a linter issue here on Ansible, which is fine to ignore. Been a long standing issue.